src/modules/session/controller/refreshToken/refreshToken.controller.ts

```diff
@@ -0,0 +1,31 @@
+import { type NextFunction, type Request, type Response } from 'express';
+import { NotAuthorizedError } from '../../../../errors';
+import { type IUser } from '../../../user/type';
+import TokenService from '../../../../services/TokenService';
+import config from '../../../../config/config';
+import UserService from '../../../user/user.service';
+
+const refreshToken = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+  try {
```

Comment:
you don't need try/catch at all. All errors will be handle by global error handler

Reply:
how will these errors be passed to this error-middleware?